config/utilities/copyFromDir.ts

```diff
@@ -0,0 +1,16 @@
+/* istanbul ignore file */
+import {execSync, exec} from 'child_process'
+/**
+ * copyFromDir
+ * @description Copies all files from source folrder to destination
+ * @param source path
+ * @param destination path
+ * @returns promise
+ */
+export const copyFromDir = async (source: string, destination: string) => {
+  // adjust trailing slash
+  const dest = `${destination.replace(/\/$/, '')}/`
+  const src = `${source.replace(/\/$/, '')}`
+  execSync(`mkdir -p ${dest}`)
+  return exec(`cp -R ${src}/* ${dest}`)
+}
```

Comment:
just curious if you'd considered these alternatives...

a) using `fs` as it has better process parallelisation 
b) running natively through a shell script, to avoiding blocking the event loop and saturating the JS thread with something like `execSync`

Reply:
No, neither of those.

I would like to keep it in the script, which is why I think **b)** is not so good. 

I am happy to do **a)** if you can point me to where I can learn how to. This is just the way I know. 🙃

Reply:
to expand on a) I think you can pass `recursive` option to `fs.cp`, which can achieve the same thing. I'm totally onboard with dropping fs-extra for something like this though.

Reply:
Wait, you mean:
- fs.readdirSync
- loop → fs.copyFile?

Reply:
`fs.cp` is experimental though, is this not a problem?

Reply:
either of the approaches you mentioned would be fine. We're at node 18 now, so not long to go to 19 where fs.cp comes in. Happy to play it safe though and use fs.copyFile instead.

Reply:
If we go with `fs.cp` do we need to add node@18 to our package.json? We currently don't have node in there. But we should maybe add it at least as a peer-dep? 🤔

Reply:
Yeah it'll run the background, but the problem there may be that you'll run into race conditions if your function returns too early. If it works, best to leave as-is. I think it's fine. Alternatively, add the await or use `copyFileSync` maybe?